infrastructure/crowdin/setup-instance.sh

```diff
@@ -8,11 +8,14 @@ PYTHON_PACKAGE_DIR="crowdin_bot_python_package"
 
 # Install packages
 sudo apt-get install git --yes
+sudo apt-get install gettext --yes
 sudo apt-get install default-jre --yes  # Java, for crowdin cli
 sudo apt-get install python3-pip --yes
 sudo apt-get install python3-lxml --yes  # Install here instead of pip3 because compilation uses too much RAM
 sudo pip3 install -U pip setuptools
```

Comment:
Could these files be stored in a requirements file to lock to same versions as the rest of the system, which will be automatically updated by PyUP with the same system.